roles/openstackclient/templates/openstack.j2

```diff
@@ -4,7 +4,7 @@
 
 PARAMETERS=${PARAMETERS:-}
 
-if [[ -t 0 ]]; then
+if [[ -t 0 && -t 1 && -t 2 ]]; then
```

Comment:
I'm not sure if the check for stderr is even needed, this might lead to issues in the opposite direction?

Reply:
I did not have a concrete problem, but thought that, e.g. piping stderr into a file would now result in a lot of non-printable characters in it.
Which issues do you have in mind?

Reply:
like what happens if someone just redirects stderr, but the command still asks for a password interactively, is this expected to work properly?

Reply:
In case of the openstack client it will fall back to a non-interactive mode when no terminal is attached and just print an error that no password ist set. I have not tested any of the other tools though